go.mod

```diff
@@ -4,14 +4,15 @@ go 1.16
 
 require (
 	cloud.google.com/go/storage v1.15.0
+	filippo.io/age v1.0.0
```

Comment:
Upgrade to latest? `v1.1.1`

Reply:
Yeah, when this was originally implemented v1.0.0 was the latest one :+1: